src/components/Navigation/ProfileMenu.tsx

```diff
@@ -0,0 +1,48 @@
+import { FaUser } from 'react-icons/fa'
+import client from '@/database/client'
+
+export default function ProfileMenu() {
+    const signOut = async () => {
+        const { error } = await client.auth.signOut()
+        if (error) throw error
+    }
+    return (
+        <div>
+            <button
+                id="dropdownDefaultButton"
```

Comment:
https://www.flowbite-react.com/docs/components/dropdown#default-dropdown

we can look at importing the react component for the dropdown to add in the functionality we're seeking here. I am thinking we were importing the flowbyte lib like a cdn in index.html to try to apply that logic here

Reply:
See above lol